ci-operator/step-registry/ipi/conf/azure/confidential/ipi-conf-azure-confidential-commands.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+PATCH="/tmp/install-config-security.yaml.patch"
+
+# OCPBUGS-36670: encryptionAtHost and settings.securityType are not related.
```

Comment:
Shall we consider to extract `encryptionAtHost` as a separate ENV var to continue all kinds of combination ?